labex/src/components/LoginPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom'
+import axios from 'axios'
+
+import { Login, ActualPage, LoginForm, LoginButton } from './styled'
+import NavBar from './NavBar'
+
+const baseUrl = "https://us-central1-labenu-apis.cloudfunctions.net/labeX/lais-mello"
+
+const LoginPage = () => {
+    
+    const [ email, setEmail ] = useState("") 
+    const [ password, setPassword ] = useState("")   
+
+    const history = useHistory();
+    
+    useEffect(() => {
+        const token = window.localStorage.getItem('token');
+        if (token !== null) {
+            history.push('/adm')
```

Comment:
Neste caso, quando o usuário já estiver logado e por algum motivo venha a acessar a página de Login, é interessante utilizar o history.replace('/adm') para evitar que o botão de retornar do navegador o mande de volta à página de Login.
```suggestion
            history.replace('/adm')
```